### PR TITLE
chore: update all the things for main

### PR DESCRIPTION
**Pull request by codyoss**

### CODE REVIEW

File: internal/gapicgen/git/github.go [hunk 1]
```diff
@@ -268,7 +268,7 @@ git push origin $BRANCH_NAME
 		Title: &t,
 		Body:  &body,
 		Head:  github.String(fmt.Sprintf("googleapis:" + gocloudBranchName)),
-		Base:  github.String("master"),
+		Base:  github.String(""),
```
Comment (noahdietz) [inline internal/gapicgen/git/github.go:271]: main?

File: pubsublite/internal/wire/message_router.go [hunk 2]
```diff
@@ -51,7 +51,7 @@ func (r *roundRobinMsgRouter) Route(_ []byte) (partition int) {
 // number. It should only be used for messages with an ordering key.
 //
 // Matches implementation at:
-// https://github.com/googleapis/java-pubsublite/blob/master/google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/internal/DefaultRoutingPolicy.java
+// https://github.com/googleapis/java-pubsublite/blob//google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/internal/DefaultRoutingPolicy.java
```
Comment (noahdietz) [inline pubsublite/internal/wire/message_router.go:54]: /main/?

File: internal/gapicgen/git/github.go (hunk 1 above)
Comment (codyoss) [reply internal/gapicgen/git/github.go]: I am not sure what happened here, good catch!

File: pubsublite/internal/wire/message_router.go (hunk 2 above)
Comment (codyoss) [reply pubsublite/internal/wire/message_router.go]: Ack

**Review by noahdietz — APPROVED** [review]

**Pull request merged**